packages/backend/src/services/UserAttributesService/UserAttributeUtils.ts

```diff
@@ -55,11 +55,9 @@ export const doesExploreMatchRequiredAttributes = (
 export const getFilteredExplore = (
     explore: Explore,
     userAttributes: UserAttributeValueMap,
-): Explore => {
+): Explore | null => {
     if (!doesExploreMatchRequiredAttributes(explore, userAttributes)) {
-        throw new AuthorizationError(
-            "You don't have authorization to access this explore",
-        );
+        return null;
```

Comment:
do you see a benefit of removing the throw here an handle the error where it's used? 🤔 can you explain the reason? If we don't want to throw an error we could catch it where we don't want that to happen. But maybe I'm not reading this correctly